Tribler/Test/test_as_server.py

```diff
@@ -212,11 +212,33 @@ def _shutdown_session(self, session):
 
         self._logger.debug("Session has shut down")
 
-    def assert_(self, boolean, reason=None, do_assert=True):
+    def assert_(self, boolean, reason=None, do_assert=True, tribler_session=None, dump_statistics=False):
```

Comment:
You could pass a reference to the tribler_session.dump_statistics method instead